src/popup/modules/UserInterface.js

```diff
@@ -712,26 +712,19 @@ export function init() {
     ]);
 }
 
-// zoom listener
-document.onkeydown = zoomQrCode;
-
 /**
  * Listens to Alt + plus/minus keypresses to alter 
  * QR code size.
- *
- * @function
- * @returns {void}
  */
-function zoomQrCode(e) {
-    var evtobj = window.event ? event : e;
-    if (evtobj.keyCode == 61 && evtobj.altKey && qrLastSize < 440) {
+document.addEventListener('keydown', function(event) {
```

Comment:
naaaw, I would still keep the function `zoomQrCode` or so you had… You can just reference that here AFAIK.